python/neuroglancer/static/__init__.py

```diff
@@ -77,6 +77,9 @@ class FileSource(StaticContentSource):
     def __init__(self, path=None, file_open=open):
         if path is None:
             path = os.path.join(os.path.dirname(__file__), '../../../dist/dev')
+
+        file_open = file_open or open
```

Comment:
None was being passed to file_open, so when it went to open the file, it died. :(